doc/UsersGuide/source/Backgroundinfo/TechnicalOverview.rst

```diff
@@ -91,3 +91,260 @@ HAFS Workflow Schematic
+    - ``./ush/setup_hurricane``: for SDM to set up hurricane message files by extracting info from the original NHC/JTWC message files in the DCOM directory.
+
+**Output:**
+    - ATCF track forecast files, 4x/day at 00Z/06Z/12Z/18Z.
+
+Locations for Output
+--------------------
+
+- **WCOSS2:**
+    - HFSA: ``/lfs/h1/ops/prod/com/hafs/v2.0/hfsa.yyyymmdd/hh``
+    - HFSB: ``/lfs/h1/ops/prod/com/hafs/v2.0/hfsb.yyyymmdd/hh``
+- **Other servers:**
+    - NCEP ftp server, NOMADS, NWS Gateway/SBN
+
+Background
+----------
+
+HAFSv2 will replace NCEP's current operational hurricane forecast systems, HAFSv1 in the 2024 hurricane season. The goal of this project is to upgrade the HAFSv1 that was running in operations in 2023. The system will provide improved and comparable tropical cyclone track and intensity forecast guidance in all global oceanic basins, as demonstrated through evaluation of retrospective and real time experiments compared against the two current configurations of operational HFSAv1 and HFSBv1.
+
+.. _scientific-updates:
+
+Scientific Changes to Improve Track and Intensity Forecast Skills
+-----------------------------------------------------------------
+
+**System and Infrastructure Upgrades:**
+    - Latest version of ``ufs-weather-model``, HAFSv2 final scientific configuration freeze on 02/08/2024.
+    - Increase moving nesting horizontal resolution from 6-2 km to 5.4-1.8 km (HFSA only).
+    - Reduce model time step from 90 to 72s (HFSB only).
+    - Updated horizontal advection options.
+    - Improved model stability and runtime efficiency.
+
+**Vortex Initialization Improvement:**
+    - Enhance vortex initialization to cycle hydrometeor variables and vertical velocity (HFSA only).
+    - Update composite vortex and reduce warm-cycling Vmax threshold from 50 to 40 kt (HFSA only).
+
+**Data Assimilation (DA) Improvement:**
+    - Ingest new high-resolution :term:`GOES-R` mesoscale :term:`AMVs`.
+    - Scale-Dependent Localization for inner core data assimilation.
+    - Refine GPS Radio Occultation (RO) DA.
+
+**Model Physics Advancement:**
+    - Upgrade Thompson :term:`MP` with bug fixes.
+    - Thompson Microphysics for :term:`NATL` basin, :term:`GFDL` Microphysics for :term:`EPAC`/:term:`CPAC` and :term:`JTWC` basins (:term:`HFSA` only).
+    - Update :term:`TKE` :term:`EDMF` :term:`PBL` and :term:`SASAS` CP schemes with vertical wind shear impacts.
+    - Change the radiation calling time step from 720s to 900s (HFSA only).
+    - Reduce radiation time step from 1800 to 720s (HFSB only).
+    - Update CO2 fix files.
+
+**Ocean/Wave Coupling:**
+    - MOM6 ocean model coupling (HFSA only).
+    - CMEPS with inline-CDEPS coupling (HFSA only).
+
+**Post-processing:**
+    - Upgrade GFDL Tracker.
+
+**Software Dependencies:**
+    - See `here <https://github.com/hafs-community/HAFS/blob/production/hafs.v2/versions/run.ver>`__
```

Comment:
```suggestion
    - See `list of dependencies here <https://github.com/hafs-community/HAFS/blob/production/hafs.v2/versions/run.ver>`_
```
The reason for the single `_` after the hyperlink is so it gives us a warning when there's more than one link with the same text. We _want_ this warning because linking on the same text reduces Section 508 compliance/makes documentation less accessible. As a general principle, it's best to avoid linking on the word "here" and instead use more descriptive words.